workers/unity/Assets/Tests/PlaymodeTests/Performance/ComponentUpdateTests.cs

```diff
@@ -0,0 +1,58 @@
+using Improbable.Gdk.Core;
+using Improbable.Gdk.TestUtils;
+using Improbable.Worker.CInterop;
+using NUnit.Framework;
+using Unity.PerformanceTesting;
+
+namespace Improbable.Gdk.PlaymodeTests
+{
+    [TestFixture]
+    public class ComponentUpdateTests : MockBase
+    {
+        private const string WorkerType = "TestWorkerType";
+        private const long EntityId = 100;
+
+        protected override MockWorld.Options GetOptions()
+        {
+            return new MockWorld.Options
+            {
+                WorkerType = WorkerType
+            };
+        }
+
+        [Performance, Test]
+        public void SinglePositionUpdate()
+        {
+            var random = new System.Random(42);
+            var markers = new[] { "GatherAllChunks", "ExecuteReplication", "Position" };
```

Comment:
We are tracking replication, but in the world we simulated receiving a component?